src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { fetchArticles } from '../utils.js';
+import ArticleCard from './ArticleCard.jsx';
+
+const ArticlesList = () => {
+    const [allArticles, setAllArticles] = useState([]);
+
+    useEffect(() => {
+        fetchArticles().then(({articles}) => {
+            setAllArticles(articles);
+        });
+    }, []);
+    return (
+        <>
+        <ul className="article-list">
+        {allArticles.map((article, index) => (<li className="article-item" key={"articlecard"+index}><ArticleCard article={article} /></li>))}
```

Comment:
As each article has an id, it's better to make use of the article_id as the unique key instead of the index